hotness/consumers.py

```diff
@@ -180,12 +180,20 @@ def handle_anitya(self, msg):
             self.log.info("Now with #%i, time to do koji stuff" % bz.bug_id)
             try:
                 # Kick off a scratch build..
-                task_id = self.buildsys.handle(package, upstream, version, bz)
+                task_id, patch_filename, description = self.buildsys.handle(
+                    package, upstream, version, bz)
+
                 # Map that koji task_id to the bz ticket we want to pursue.
                 self.triggered_task_ids[task_id] = bz
-            except Exception:
-                self.log.warning("Failed to kick off scratch build.")
+
+                # Attach the patch to the ticket
+                self.bugzilla.attach_patch(patch_filename, description, bz)
+            except Exception as e:
+                heading = "Failed to kick off scratch build."
+                note = heading + "\n\n" + str(e)
+                self.log.warning(heading)
```

Comment:
Not necessarily.  The whole exception traceback gets logged on the very next line.